packages/server/tests/acceptance/data/conformity/utils/validations.ts

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: Apache-2.0
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+import { expect } from 'chai';
+
+import { ErrorResponse, JsonRpcResponse, Method, Schema } from './interfaces';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const execApisOpenRpcData = require('../../../../../../../openrpc_exec_apis.json');
```

Comment:
Seems like this file doesn't exist yet when the CI is ran and it is causing it to fial, perhaps you can load it lazily here?